source/databricks/package/schemas/time_series_points_schema.py

```diff
@@ -11,27 +11,28 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from package.codelists import Colname
 from pyspark.sql.types import (
     DecimalType,
     StructType,
     StructField,
     StringType,
     TimestampType,
     IntegerType,
+    LongType,
 )
 
 time_series_points_schema = StructType(
     [
-        StructField(Colname.metering_point_id, StringType(), True),
-        StructField(Colname.transaction_id, StringType(), True),
-        StructField(Colname.quantity, DecimalType(18, 3), True),
-        StructField(Colname.quality, IntegerType(), True),
-        StructField(Colname.time, TimestampType(), True),
-        StructField(Colname.resolution, IntegerType(), True),
-        StructField(Colname.year, IntegerType(), True),
-        StructField(Colname.month, IntegerType(), True),
-        StructField(Colname.day, IntegerType(), True),
-        StructField(Colname.registration_date_time, TimestampType(), True),
+        StructField("MeteringPointId", StringType(), True),
+        StructField("TransactionId", StringType(), True),
+        StructField("Quantity", DecimalType(18, 3), True),
+        StructField("Quality", LongType(), True),
+        StructField("Resolution", LongType(), True),
```

Comment:
Why do you use a larger type here?
```suggestion
        StructField("Quality", IntegerType(), True),
        StructField("Resolution", IntegerType(), True),
```

Reply:
LongType is what it defaults to when not given a specific schema.

Reply:
time_series_point_schema has been removed, since it was not used